addon/components/select-input.js

```diff
@@ -0,0 +1,40 @@
+import Input from './abstract-input'
+import * as listUtils from '../list-utils'
+import Ember from 'ember'
+
+export default Input.extend({
+  classNames: [
+    'frost-bunsen-input-select',
+    'frost-field'
+  ],
+
+  init () {
+    this._super(...arguments)
+    this.set('options', Ember.A([]))
+  },
+
+  dbStore: Ember.inject.service('store'),
+
+  selectedOptions: [],
+
+  didReceiveAttrs () {
+    this._super(...arguments)
+    const modelDef = this.get('model')
+    if (modelDef) {
```

Comment:
I'd change this to:
```javascript
if (!modelDef) {
  return
}
```
so the remaining logic doesn't need to be indented as much